### PR TITLE
Added Session Accept/Reject view

### PR DESCRIPTION
**Pull request by aditya1702**
#504 
session1
And in the accepted sessions:
session2

### CODE REVIEW

**Comment by aditya1702** [comment]
@SaptakS @rafalkowalski Done!Can be merged

File: open_event/views/admin/models_views/session.py [hunk 1]
```diff
@@ -21,3 +21,17 @@ def display_view(self, event_id):
...
                            sessions=sessions)
+
+    @expose('/<int:session_id>/accept_session', methods=('GET', 'POST'))
+    def accept_session(self, event_id, session_id):
+        session = DataGetter.get_session(session_id)
+        session.state = 'accepted'
+        save_to_db(session, 'Session Accepted')
+        return redirect(url_for('.display_view', event_id=event_id))
+
+    @expose('/<int:session_id>/reject_session', methods=('GET', 'POST'))
```
Comment (rafalkowalski) [inline open_event/views/admin/models_views/session.py]: Only POST method, remove GET

File: open_event/views/admin/models_views/session.py [hunk 2]
```diff
@@ -21,3 +21,17 @@ def display_view(self, event_id):
         sessions = DataGetter.get_sessions_by_event_id(event_id)
         return self.render('/gentelella/admin/session/display.html',
                            sessions=sessions)
+
+    @expose('/<int:session_id>/accept_session', methods=('GET', 'POST'))
```
Comment (rafalkowalski) [inline open_event/views/admin/models_views/session.py]: Only POST method, remove GET

**Comment by aditya1702** [comment]
@rafalkowalski Done1

**Comment by rafalkowalski** [comment]
@aditya1702 Are you sure that it's working?

**Comment by aditya1702** [comment]
@rafalkowalski Yes :grin:

**Pull request merged**